packages/ui/src/utils/components.tsx

```diff
@@ -12,8 +12,9 @@ if (theme) {
 
 const twMergeWithConfig = extendTailwindMerge({
     prefix: "cui-",
-    theme,
-    classGroups,
+    extend: {
```

Comment:
Is theme not needed anymore?

Reply:
I tried to start up the frontend locally and didn't notice any difference really. I'm also not noticing any difference in the UI preview above.